scripts/tests/provisioning/configure_vm.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
```

Comment:
I feel like this script does much more than configuring a VM, and would benefit from a rename.

Reply:
Maybe `start-vm-using-expect.sh`?